src/07-projects/project-items.apib

```diff
@@ -71,4 +71,43 @@ Get a list of project items.
                 + updated_at: `2018-12-05T16:44:33+00:00` (string)
         + meta (object)
             + page (Pagination)
-            + matches: `120` (number)
\ No newline at end of file
+            + matches: `120` (number)
+
+### projectItems.info [GET /projectItems.info]
+
+Get details for a single project item.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `8a04371b-2ffb-407b-9b24-d5b5452009c7` (string, required)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `cfb4146d-06be-41f1-bb39-aa3c929c71dc` (string)
+            + project (object)
+                + type: `project` (string)
+                + id: `944534fb-15f1-4eea-aab1-82a427aa2d0d` (string)
+            + milestone (object, nullable)
+                + type: `milestone` (string)
+                + id: `944534fb-15f1-4eea-aab1-82a427aa2d0d` (string)
+            + type: `tracked_time` (enum[string])
+                + Members
+                    + tracked_time
+                    + external_cost_item
+            + description: `Drafted the document` (string, nullable)
+            + occurred_at: `2018-11-26T16:44:33+00:00` (string)
```

Comment:
To me it this still sounds a bit weird. But if we can't find any other name, let's go for this 👌